### ISSUE TITLE
FR: Can obsidian tool panel set shortcut keys? 

### ISSUE THREAD

**Issue by zhl111** [zsviczian/obsidian-excalidraw-plugin#1940]
![image](https://github.com/user-attachments/assets/ab3b3890-82a8-45dd-bd51-5f8d835443ee)

Can you set shortcut keys to hide this panel? I want to close the whole panel, not the folding panel
![image](https://github.com/user-attachments/assets/516c910e-f016-4dc0-a60c-8b6f4eb03b46)

**Comment by zsviczian** [comment]
yeah, this has been on my mind for a long time. Would be good, also to add shortcuts for pens.

**Comment by LeonelRFF** [comment]
Excellent proposal. Also, it would be handy if the tool panel had a larger location in the sidebar, rather than being floating, similar to when you click on the library icon and a side panel with all the drawings is displayed. This would be especially useful when working on tablets or iPads, considering that new scripts are continually being added.
Regards.

**Comment by dimateos** [comment]
I was looking for this too as a quick way to use scripts grouped by folders etc.
IMO It would also help with not being able to pin default tools https://github.com/zsviczian/obsidian-excalidraw-plugin/issues/1100., but probably might still not be ideal for some users.

Tried implementing a hotkey for the tools panel myself following https://github.com/zsviczian/obsidian-excalidraw-plugin/issues/2338, but there is no appstate change it seems.

Btw just wanted to reference an issue I open some time ago about pen hotkeys as you mentioned it https://github.com/zsviczian/obsidian-excalidraw-plugin/issues/2488.
I guess pens also dont cange the appstate like the tools panel etc.

Best regards
